internal/sirius/allocate_cases_test.go

```diff
@@ -0,0 +1,100 @@
+package sirius
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/pact-foundation/pact-go/dsl"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAllocateCases(t *testing.T) {
+	t.Parallel()
+
+	pact := newPact()
+	defer pact.Teardown()
+
+	testCases := []struct {
+		name          string
+		setup         func()
+		cookies       []*http.Cookie
+		expectedError func(int) error
+		file          *NoteFile
+	}{
+		{
+			name: "OK",
+			setup: func() {
+				pact.
+					AddInteraction().
+					Given("I have a pending case assigned").
+					UponReceiving("A request to change the assignee of the case").
```

Comment:
Probably worth doing a test with multiple cases being assigned at once, since that's an important (and weird) bit of the contract).